viewer/packages/tools/src/Overlay3D/Overlay3DTool.ts

```diff
@@ -44,6 +44,26 @@ export type Overlay3DToolParameters = {
   defaultOverlayColor: THREE.Color;
 };
 
+export type OverlayCollectionOptions = {
+  /**
+   * Sets default overlay color for newly added overlays.
+   * Default is yellow.
+   * */
+  defaultOverlayColor?: THREE.Color;
+  /**
+   * Sets default texture for all overlays of this OverlayCollection.
+   * Must be a square texture with size at least as `maxPointSize`.
+   * Texture should be monochrome, internally, R channel is used as a mask for denoting pixels
+   * that should be colored by texture and not by overlay color.
+   */
+  overlayTexture?: THREE.Texture;
+  /**
+   * Sets whether overlays are always displayed as circles. Regardless of
+   * texture used. Default is true.
+   */
+  circularOverlay?: boolean;
```

Comment:
I still don't completly understand the need for this when we have a masking texture...

Reply:
Since alpha is non zero on the very edge of the texture you still see color of the overlay "spilling" outside of white border, so that's the reason for adding this cut. But also when I think about it after demoing texture support, it could be quite useful in general. Most of the images are squared, so adding capability to easily make it circular and look nice is a nice feature to have.
![Screenshot 2023-05-26 at 16 48 54](https://github.com/cognitedata/reveal/assets/45129444/94938be5-d10e-4e3a-b2b3-5ddb2432769b)
![Screenshot 2023-05-26 at 16 53 25](https://github.com/cognitedata/reveal/assets/45129444/781a1bf1-f878-4d33-9bac-c5e74a3f238f)

Reply:
Removed it.